setup.py

```diff
@@ -297,6 +297,41 @@ def append_nvcc_threads(nvcc_extra_args):
         )
     )
 
+    # Check, if CUDA11 is installed for compute capability 8.0
+    cc_flag = []
+    _, bare_metal_major, _ = get_cuda_bare_metal_version(CUDA_HOME)
+    if int(bare_metal_major) >= 11:
```

Comment:
Would we also want to include `sm_86`?

Reply:
done in 5100286a18afb58e10fccc491642f9e4990ad75b